File: 백준/Bronze/10813. 공 바꾸기/공 바꾸기.java
import java.io.*;
import java.util.Arrays;
import java.util.StringTokenizer;

public class Main {

    public static void main(String[] args) throws IOException {

    BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
    BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(System.out));
    StringTokenizer st = new StringTokenizer(br.readLine(), " ");

    int N = Integer.parseInt(st.nextToken());
    int M = Integer.parseInt(st.nextToken());

        int[] arr = new int[N];

        for (int x = 0; x < N; x++){
            arr[x] = x+1;
        }

        for (int y = 0; y < M; y++){
            st = new StringTokenizer(br.readLine(), " ");
            int i = Integer.parseInt(st.nextToken());
            int j = Integer.parseInt(st.nextToken());
            int temp_i = arr[i-1];
            int temp_j = arr[j-1];

            arr[i-1] = temp_j;
            arr[j-1] = temp_i;
        }

        for (int a : arr){
            System.out.print(a +" ");
        }
    }
}

File: 백준/Bronze/10813. 공 바꾸기/공 바꾸기.py
import sys

N, M = map(int, sys.stdin.readline().split())

arr = [x for x in range(1, N+1)]  # [1,2,3,4,5]

for _ in range(M):
    i, j = map(int, sys.stdin.readline().split())
    arr[i-1], arr[j-1] = arr[j-1], arr[i-1]

for k in range(N):
    print(arr[k], end=" ")